### DISCUSSION TITLE
How do I restore the default properties of a Control?

### DISCUSSION THREAD

**Discussion by kyuranger** [AvaloniaUI/Avalonia#16874] [Q&A] [answered]
In the source code of PathIcon(https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/PathIcon.xaml):

Height and Width are limited to IconElementThemeHeight and IconElementThemeWidth respectively, which are 20px on Windows. But I want it to be able to be scaled smaller, so I want to restore it to the default Double.NaN. But I don't know how to write the Style to do this. Please tell me the solution, thank you.

BTW, it seems WinUI can set Auto or double in XAML to restore it directly while Avalonia cannot do this:
https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.frameworkelement.width?view=windows-app-sdk-1.5

**Comment by rabbitism** [comment, marked as answer]
If you need double.NaN you can set it to `{x:Static x:Double.NaN}`

**Reply by kyuranger** [reply]
It works. Thank you.
